src/Shmuelie.WinRTServer/Windows/Com/IActivationFactory.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.Marshalling;
+using Shmuelie.WinRTServer.Windows.Com.Marshalling;
+
+namespace Shmuelie.WinRTServer.Windows.Com;
+
+[GeneratedComInterface]
+[Guid("00000035-0000-0000-C000-000000000046")]
+internal unsafe partial interface IActivationFactory : IInspectable
+{
+    [PreserveSig]
+    [return: MarshalUsing(typeof(HResultMarshaller))]
+    global::Windows.Win32.Foundation.HRESULT ActivateInstance(void** instance);
```

Comment:
There's a helper API for this in CsWinRT by the way, in case it's useful.

Reply:
What's the type?

Reply:
I think Sergio means this:
![image](https://github.com/user-attachments/assets/c5d3e342-4c97-4e4e-a1fc-e98db65b71bc)
`WinRT.Interop.IActivationFactory`

Reply:
Using it would be complicated, plus changing it wouldn't be a breaking change so I won't block the 2.0 release on it.